R/teal_data-show.R

```diff
@@ -0,0 +1,19 @@
+#' Show `teal_data` object
+#'
+#' Prints `teal_data` object
+#' @param object (`teal_data`)
+#' @return Input `teal_data` object.
+#' @importFrom methods show
+#' @examples
+#' teal_data()
+#' teal_data(x = iris, code = "x = iris")
+#' verify(teal_data(x = iris, code = "x = iris"))
+#' @export
+setMethod("show", "teal_data", function(object) {
+  if (object@verified) {
+    cat("✅︎", "verified teal_data object\n")
+  } else {
+    cat("✖️︎", "unverified teal_data object\n")
```

Comment:
Use utf encoding to pass checks

```suggestion
    cat("\u2705\ufe0e", "verified teal_data object\n")
  } else {
    cat("\u2716", "unverified teal_data object\n")
```

Reply:
Ah thanks, was hoping to figure this out on my own at some point